application/views/templates/admin/template.php

```diff
@@ -13,14 +13,28 @@
 		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
 		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
 
-    	<script src="//code.jquery.com/jquery-3.2.1.min.js"></script>
+    <script src="//code.jquery.com/jquery-3.2.1.min.js"></script>
 		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
 
 		<link rel="stylesheet" type="text/css" href="<?php echo base_url(); ?>assets/go/css/go_normalize.css">		
 		<link rel="stylesheet" type="text/css" href="<?php echo base_url(); ?>assets/go/css/go_default.css">	
+		
+		
+		<script type="text/javascript">
```

Comment:
I like that you put this in the bottom of the footer.php file, but can you merge the JS from the top of the file down to yours so that it's all together?  Also, we don't need 2 .on("load") so merge their contents into one.

Also, something is wrong with your indentation.  Please make sure your IDE settings for indentation match mine.  Thanks,

![indent](https://user-images.githubusercontent.com/10435081/29190630-87d67a80-7de0-11e7-8669-e30812c0d011.png)

![2](https://user-images.githubusercontent.com/10435081/29190648-913a432c-7de0-11e7-8f4b-ab4d50196e54.png)